### PR TITLE
add brand icon and logo for integration

### PR DESCRIPTION
**Pull request by donfrensis**
## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [X] Add a new logo or icon for a custom integration (custom component)
  - [X] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 
- Link to custom integration repository: https://github.com/donfrensis/HA-IntesisHome-Local 

## Checklist

- [X] The added/replaced image(s) are **PNG**
- [X] Icon image size is 256x256px (`icon.png`)
- [ ] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [X] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Hi there @donfrensis 👋 

Thanks for the PR, unfortunately, we cannot accept this branding, as it is a modified version of the original branding of the manufacturer. These kind of modifications are most often not allowed in the branding guidelines most manufacturers have. Therefore, we don't accept them.

Please use the original branding of the manufacturer instead.

Thanks! 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]
Ooops wrong review button, I wanted to request changes. Let me try again...

**Comment by frenck** [comment]
Thanks for the pushes @donfrensis 👍 

Don't forget to mark the PR ready for review using the "Ready for review" button once you are done and happy with your changes.

../Frenck

**Comment by donfrensis** [comment]
sorry! it's my first time!

**Review by donfrensis — COMMENTED** [review]
Replaced icon.png and logo.png as requested

**Review by frenck — APPROVED** [review]
Thanks, @donfrensis 👍 


../Frenck

**Comment by donfrensis** [comment]
Thanks!
